typedoc.json

```diff
@@ -0,0 +1,10 @@
+{
+  // https://typedoc.org/options/
+  "$schema": "https://typedoc.org/schema.json",
+  "entryPoints": ["fabric.ts"],
```

Comment:
```suggestion
   "entryPoints": ["index.ts", "index.node.ts"],
```

support node entry?

Reply:
let's start with one and let's see how it works